workflow/pipeline/photometry.py

```diff
@@ -53,79 +53,262 @@ class EmissionColor(dj.Lookup):
+                }
+            )
+
+            # Populate FiberPhotometry
+            for trace_name in trace_names:
+
+                # Populate EmissionColor if present
+                emission_color = color_mapping[trace_name.split("_")[1][0]]
+
+                emission_wavelength = (
+                    meta_info.get("Fiber", {})
+                    .get("emission_wavelength", {})
+                    .get(emission_color, None)
+                )
+
+                EmissionColor.insert1(
+                    {
+                        "emission_color": emission_color,
+                        "wavelength": emission_wavelength,
+                    },
+                    skip_duplicates=True,
+                )
+
+                # Populate SensorProtein if present
+                sensor_protein = (
+                    meta_info.get("VirusInjection", {})
+                    .get(hemisphere, {})
+                    .get("sensor_protein", None)
+                )
+                if sensor_protein:
+                    SensorProtein.insert1(
+                        {"sensor_protein_name": sensor_protein}, skip_duplicates=True
+                    )
+
+                # Populate ExcitationWavelength if present
+                excitation_wavelength = (
+                    meta_info.get("Fiber", {})
+                    .get("excitation_wavelength", {})
+                    .get(emission_color, None)
+                )
+
+                if excitation_wavelength:
+                    ExcitationWavelength.insert1(
+                        {"excitation_wavelength": excitation_wavelength},
+                        skip_duplicates=True,
+                    )
+
+                demodulated_trace_list.append(
+                    {
+                        **key,
+                        "fiber_id": fiber_id,
+                        "hemisphere": hemisphere,
+                        "trace_name": trace_name.split("_")[0],
+                        "emission_color": emission_color,
+                        "sensor_protein_name": sensor_protein,
+                        "excitation_wavelength": excitation_wavelength,
+                        "demod_sample_rate": demod_sample_rate,
+                        "trace": photometry_df[
+                            trace_name + hemisphere[0].upper()
+                        ].values,
+                    }
+                )
+
+        # Populate FiberPhotometry.Fiber
+        self.Fiber.insert(fiber_list)
+        # Populate FiberPhotometry.DemodulatedTrace
+        self.DemodulatedTrace.insert(demodulated_trace_list)
+
+
+@schema
+class FiberPhotometrySynched(dj.Imported):
+    definition = """
+    -> FiberPhotometry
+    ---
+    timestamps   : longblob
+    time_offset  : float     # time offset to synchronize the photometry traces to the master clock (in second)  
+    sample_rate  : float     # target downsample rate of synched data (in Hz) 
+    """
+
+    class SynchedTrace(dj.Part):
```

Comment:
`Synched` or `Synced` ?

Reply:
I think both are fine but will change to synced for brevity